client/src/Tools/FetchData/fetchContestsfunctions.js

```diff
@@ -0,0 +1,16 @@
+import { requestOptionsGET } from './requestOptions';
+
+export async function getContestsCards(taker) {
+  let url = '/api/contests/card/data';
+  if (taker) {
+    url = `/api/contests/card/data?taker=${taker}`;
+  }
+
+  const response = await fetch(url, requestOptionsGET);
+  if (response.status === 200) {
+    const result = await response.json();
+    return result;
+  } else {
```

Comment:
What happens when there is no contests? Redirect to error page?

Reply:
Good point, I talked yesterday with Tomek to make Error Page as compontent which will be rendered instead of content, when there will be problem with getting data. So maybe we can do that topic as TODO when error page / error component will be able to use? Will be easier to consider options